runtime/jcl/unix/syshelp.c

```diff
@@ -254,16 +254,20 @@ char *getPlatformFileEncoding(JNIEnv * env, char *codepageProp, int propSize, in
 #if defined(J9ZTPF)
     return "ISO8859_1";
 #endif /* defined(J9ZTPF) */
-	PORT_ACCESS_FROM_ENV(env);
-	char *codepage;
-	int i, nameIndex;
+	char *codepage = NULL;
+	int i = 0;
+	int nameIndex = 0;
 #if defined(LINUX)
-    IDATA result;
-    char langProp[24], *ctype;
-#endif
+    IDATA result = 0;
```

Comment:
since you've modified this code, can you fix the indent here?

Reply:
fixed.